queue_services/entity-emailer/requirements.txt

```diff
@@ -7,10 +7,13 @@ asyncio-nats-streaming==0.4.0
 attrs==19.1.0
 blinker==1.4
 certifi==2020.4.5.1
+chardet==3.0.4
 click==7.1.2
+idna==2.9
 itsdangerous==1.1.0
```

Comment:
are we using this?

Reply:
I think it's a dependency for something else (it gets generated off the build-req when installing the packages in prod.txt, etc.)